test/functionalTest/cases/2015_notification_templates/notification_templates_get_subscription.test.DISABLED

```diff
@@ -1,42 +0,0 @@
-# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Yes, GET /v2/subscriptions is used in notification_templates_cache_refresh.test, step 05 (missing in TOC ;-))
Also used in notification_templates_with_decoded_chars_in_notifications.test